### DISCUSSION TITLE
Allow habits to have multiple reminders

### DISCUSSION THREAD

**Discussion by that0n3guy** [iSoron/uhabits#179] [Feature Requests]
This is not the same as: https://github.com/iSoron/uhabits/issues/121 as I'm not looking to log a habit "check" more than once.  I just want to be reminded more than once.

Rewire has the feature to do multiple reminders (at different times of the day) per habit.   I use this on almost every habit I have.

Use case:
- Workout 3x per week: In Rewire, I have a reminder popup at 8am, if I worked out in the morning that day, I click the "done" button.   If I don't workout in the morning, I may do it in the afternoon so I have another reminder popup around 4.  

Without 2 reminders, I will feel like I need to leave the notification up on my phone all day so I remember to mark it in the afternoon (which is annoying) OR I just forget to mark it.

Use case 2:
- I set 2 reminders.  The first one I use as a "remember to actually do it" reminder.  The second I use as a "remember to log that you did it, or failed (see:https://github.com/iSoron/uhabits/issues/96) at it"

**Comment by sciamano** [comment]
may be you will also find interesting my ideas on integration with opentasks,
see https://github.com/dmfs/opentasks/issues/269

**Comment by jtrollkarl** [comment]
This can already be accomplished, but in a slightly different way. You can change the snooze timer from the default value of 15 mins all the way up to 24 hours. Though this only works for ALL habits, and cannot be assigned specifically. I'm currently looking in to implementing this feature on a per habit basis though.

**Comment by DarkSphere00** [comment]
Hello, are there any news on this one?

I'm trying to use Habits to improve my study performance, and it would be very much helpful to be able to set multiple reminders of a single habit, instead of splitting the habit into two (or more) different ones.
This is due to the fact that I can't always study the same subject at a single pre-determined time every day of the week, but still it would be essential to have the success rate and other data all together.

As an example, I set myself to study Mathematical analysis four times a week, two days a week at 7:45 am and two other days at 5:15 pm: at the moment, to accomplish so with Habits I need to create two different tasks, which partially defeats the purpose (and for other subjects I would need three or more tasks due to how I've organized them).

Simply having a feature to add more than one reminder would completely solve this problem...

**Comment by 3BD-R** [comment]
having multiple reminders per day for one habit is soo good and helping in usage

**Reply by karanrajpal14** [reply]
I agree! Is there a chance this would be considered @iSoron? I have several numerical habits that need to happen x times a day and sometimes I just forget about them. I'd like to be reminded every hour or so to make sure that I don't lose track. Thank you!

**Comment by snippet78** [comment]
I use the later (or snooze) option to remind me when I postpone a habit to later in the day

**Comment by jocelynthode** [comment]
This would be a nice feature!

**Comment by salomaoparkour** [comment]
Hey, it's been 7 years. This is still a desired feature 🙂

I already use it on another app and I'm trying to bring everything to Loop. But this missing feature is making my decision more difficult =(

**Comment by BrendonSzabo** [comment]
Hello, is this feature still in consideration? I would love to have this.

**Reply by AdityaMH** [reply]
Sadly, no

**Comment by AdityaMH** [comment]
Habstick has better multiple reminder.

**Comment by hockeymikey** [comment]
This is a must have IMO. Should help building those habits too.
